### PR TITLE
Fixes #9382 - Use connection info when connecting to docker

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

**Comment by eLobato** [comment]
@daviddavis Is this PR meant to wipe out fog from the code base?

**Comment by daviddavis** [comment]
@eLobato I was hoping to get by without doing that but I think I'll have to. So I haven't addressed it but I will.

**Comment by daviddavis** [comment]
@eLobato so this should take care of most of the fog stuff. Not sure how to handle the poweroff/reset/etc stuff since I don't see where it's being called. Not too familiar with the compute resource stuff either.

**Comment by eLobato** [comment]
@daviddavis :+1: , can you change `get_logs` and `get_processes` by just `logs` `processes` (as it is right now)? Otherwise I haven't found issues.

**Comment by eLobato** [comment]
Forget about @houndci , it's using the wrong rubocop.yml

**Comment by daviddavis** [comment]
@eLobato want to test this out and re-review?

**Comment by daviddavis** [comment]
I think this might be a candidate for 1.2.1?

**Comment by eLobato** [comment]
@daviddavis yup, testing.

**Comment by eLobato** [comment]
@daviddavis Still getting some errors when even just one of the Docker compute resources tabs in Containers index authentication is wrong. Like this:



Sorry, I think this will have to miss 1.2.1 but I'm all for keep on releasing weekly so next week it could be in.

**Comment by daviddavis** [comment]
@eLobato I just rebased which should fix the problem. Can you retest?

**Comment by daviddavis** [comment]
@eLobato I think it should be fixed now. Test it out and let me know. TIA.

**Comment by daviddavis** [comment]
@eLobato ping?

File: app/views/containers/show.html.erb [hunk 1]
```diff
@@ -108,7 +107,9 @@
         </div>
       </div>
       <div class="tab-pane" id="logs">
-        <pre class="terminal terminal-output pre-scrollable"><code><%= Docker::Container.get(@container.uuid).logs(:stdout => true, :tail => 100) %></code></pre>
+        <pre class="terminal terminal-output pre-scrollable"><code>
+          <%= logs(@container.uuid, :stdout => true, :tail => 100) %>
```
Comment (eLobato) [inline app/views/containers/show.html.erb]: `logs` is called here using the uuid of the container. However the method expects the Container object. 

It results on `undefined method compute_resource for #<String:0x007f2e9aa83980>` on Container#show on a container that is on.

**Comment by eLobato** [comment]
@daviddavis Container#show doesn't work currently, other than that :+1: . 
It gives me the chills a bit to merge the change, perhaps a sign our tests are not so trustable... :smile:

**Comment by daviddavis** [comment]
@eLobato I feel that way [every time I look at a PR](https://github.com/theforeman/foreman-docker/pull/93#issuecomment-77621532). Will look at containers#show.

**Comment by daviddavis** [comment]
@eLobato fixed.

**Comment by eLobato** [comment]
@daviddavis I'm going to test this now, any clue why HoundCI decided not to read `https://github.com/theforeman/foreman-docker/blob/master/.hound.yml` ?

**Comment by daviddavis** [comment]
I think those comments were from before I rebased with master.

**Comment by eLobato** [comment]
Merged as 5397ee73e891b8c9b9a2cda87a26954982f59133 , thanks @daviddavis  :fearful:

**Comment by daviddavis** [comment]
:scream:

**Pull request closed without merging**